### PR TITLE
Core WG call notes from Jan 28th

### PR DESCRIPTION
**Pull request by brghena**
### Pull Request Overview

Notes from Core Working Group call, January 28th 2022.

[Rendered](https://github.com/tock/tock/blob/b26b08cb4f6735dbc9c188ba947031d6861b3402/doc/wg/core/notes/core-notes-2022-01-28.md)

### Testing Strategy

Skimming and spellcheck.

### TODO or Help Wanted

Make sure I didn't grossly misquote you.

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make prepush`.

### CODE REVIEW

**Review by lschuermann — DISMISSED** [review]
Thanks @brghena!

File: doc/wg/core/notes/core-notes-2022-01-28.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+- Johnathan Van Why
+- Alyssa Haroldsen
+- Brad Campbell
+- Alexandru Radovici
+- Amit Levy
+- Jett Rink
+
+## Updates
+
+### uformat library
```
Comment (hudson-ayers) [inline doc/wg/core/notes/core-notes-2022-01-28.md:19]:
```suggestion
### ufmt library
```

File: doc/wg/core/notes/core-notes-2022-01-28.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+ * Hudson: On uFormat stuff. I've completely implemented width specifiers, so you can pad numbers with zeros or any type with spaces, at least on the right side of the type. Left spacing for non-numeric types would take a re-architecture. I've gotten the size overhead down to like 1.5 KB out of 13 KB for the applications I'm looking at. So now they only save 11.5 KB by changing to this uFormat solution. Still a pretty big win.
```
Comment (hudson-ayers) [inline doc/wg/core/notes/core-notes-2022-01-28.md:20]:
```suggestion
 * Hudson: On uFmt stuff. I've completely implemented width specifiers, so you can pad numbers with zeros or any type with spaces, at least on the right side of the type. Left spacing for non-numeric types would take a re-architecture. I've gotten the size overhead down to like 1.5 KB out of 13 KB for the applications I'm looking at. So now they only save 11.5 KB by changing to this uFmt solution. Still a pretty big win.
```

File: doc/wg/core/notes/core-notes-2022-01-28.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+ * Alyssa: Does uformat support lowercase x or uppercase X or both? Would only one save a non-trivial amount of space?
```
Comment (hudson-ayers) [inline doc/wg/core/notes/core-notes-2022-01-28.md:21]:
```suggestion
 * Alyssa: Does ufmt support lowercase x or uppercase X or both? Would only one save a non-trivial amount of space?
```

**Review by hudson-ayers — APPROVED** [review]

**Review by hudson-ayers — APPROVED** [review]

**Review by jrvanwhy — APPROVED** [review]

**Review by lschuermann — APPROVED** [review]

**Pull request merged**